R/cran.R

```diff
@@ -7,14 +7,15 @@
 #' other packages, you also need to describe the steps you have taken to
 #' ensure that it does not break existing code on CRAN.
 #'
+#' @param open Open the new file in the text editor. Defaults to TRUE.
```

Comment:
It's better to not copy and paste documentation and instead document in one place and use `@inheritParams()`.